itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/adapter/openshift/CDKImageRegistryUrlDiscoveryFailureTest.java

```diff
@@ -37,23 +40,35 @@ protected String getServerAdapter() {
 		return SERVER_ADAPTER_32;
 	}
 	
+	@BeforeClass
+	public static void setupCDKImageRegistrUrlDiscovery() {
+		setupOCForWorkspace();
+	}
+	
+	@Before
+	public void setupOpenshiftConnectionWizard() {
+		wizard = getOpenshiftConnectionWizard(findOpenShiftConnection(null, OPENSHIFT_USERNAME));
+		assertTrue(wizard.getAuthSection().getMethod().equals(AuthenticationMethod.BASIC));
+		switchOffPasswordSaving(wizard);
+	}
```

Comment:
@odockal this code is duplicated in 3 classes, could this be moved to one method or in abstract class?